### PR TITLE
gcc 11.2.0

### PR DESCRIPTION
**Pull request by bevanjkay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/gcc.rb [hunk 1]
```diff
@@ -9,12 +9,11 @@ class Gcc < Formula
     sha256 "ce862b4a4bdc8f36c9240736d23cd625a48af82c2332d2915df0e16e1609a74c"
     version "11.1.0"
```
Comment (carlocab) [inline Formula/gcc.rb:10]:
```suggestion
    version "11.2.0"
```

Comment (bevanjkay) [reply Formula/gcc.rb:10]: @carlocab this is really strange because I used bump-formula-pr. Is it better to close and open again?

Comment (carlocab) [reply Formula/gcc.rb:10]: Just edit this -- `bump-formula-pr` doesn't work for formulae like this one.

**Comment by carlocab** [comment]
CC @fxcoudert in case you want to and have time to update the ARM branch.

**Comment by carlocab** [comment]
Linux failures look like the ones from #81802. CC @Homebrew/linux

There is a [linkage failure](https://github.com/Homebrew/homebrew-core/pull/82045/checks?check_run_id=3182816690#step:10:4669) for `mesa`, though:
```
==> brew linkage --test mesa
==> FAILED
Missing libraries:
  unexpected (/home/linuxbrew/.linuxbrew/lib/libelf.so.0)
```
I think this is just from `test-bot` doing `brew unlink libelf` [earlier in the test run](https://github.com/Homebrew/homebrew-core/pull/82045/checks?check_run_id=3182816690#step:10:3755).

**Comment by iMichka** [comment]
`elfutils` is also involved. Both `elfutils` and `libelf` conflict, so there might be a subtle test-bot issue here.

`elfutils` is only used by `kcov` and `sysdig` so it might be worth looking into using `libelf` there too, as we use `libelf` everywhere.

**Review by carlocab — APPROVED** [review]
I think the Linux failures are unrelated, so this should be good to merge.

**Pull request closed without merging**